### DISCUSSION TITLE
Abnormally high CPU usage with 1.7.4273

### DISCUSSION THREAD

**Discussion by vaibhavgarg1982** [community/community#8928] [Copilot Conversations]
The CPU usage by the extension host process of VS code is abnormally high after update to 1.7.4273. 
This issue is resolved after rolling back to 1.7.3862.

The conversation and details are in the linked issues.

[Original issue](https://github.com/microsoft/vscode/issues/139164#issuecomment-995444851_)

[Resolution](https://github.com/microsoft/vscode/issues/139164#issuecomment-995448465)

**Comment by Krzysztof-Cieslak** [comment]
This is already tracked and discussed in https://github.com/github/feedback/discussions/8861
